src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -8,10 +8,9 @@
   </PropertyGroup>
 
   <!-- Mobile and legacy targets are disabled by default to speed up dev box -->
-  <PropertyGroup Condition="'$(INCLUDE_MOBILE_AND_LEGACY_TFM)' != ''">
+  <PropertyGroup Condition="'$(INCLUDE_MOBILE_AND_LEGACY_TFM)' != '1'">
```

Comment:
Revert 1.
Rename to `INCLUDE_MOBILE_TFM` maybe?

Reply:
Done "1". I wasn't able to find `INCLUDE_MOBILE_AND_LEGACY_TFM` in the yaml files. I think it's simpler to just leave it as is.